pip.md

```diff
@@ -40,20 +40,25 @@ RAPIDS users can once again install RAPIDS via pip!  This is an **experimental r
 
 > <i class="fas fa-desktop text-white"></i> **OS:** One of the following OS versions:
 
->> <i class="fa-brands fa-ubuntu text-white"></i> Ubuntu 18.04/20.04 or CentOS 7 / Rocky Linux 8 with <code>gcc/++</code> 9.0+
-
->> <i class="fas fa-desktop text-white"></i> Windows 11 using WSL2  **[See separate install guide <i class="fa fa-angle-double-right" aria-hidden="true"></i>](wsl2.html){: target="_blank"}**
-
+>> <i class="fa-brands fa-ubuntu text-white"></i> Ubuntu 18.04/20.04 or CentOS 7 / Rocky Linux 8 with <code>gcc/++</code> 9.0+  <br>
+>> <i class="fas fa-desktop text-white"></i> Windows 11 using WSL2  **[See separate install guide <i class="fa fa-angle-double-right" aria-hidden="true"></i>](wsl2.html){: target="_blank"}** <br>
 >>> <i class="fas fa-chevron-circle-right text-white"></i> In addition, WSL2 pip installations require following **[this guide to install the CUDA Toolkit without drivers.](https://docs.nvidia.com/cuda/wsl-user-guide/index.html#cuda-support-for-wsl2){: target="_blank"}**
 
-> <i class="fas fa-info-circle text-white"></i> **Glibc version:** x86_64 wheels require glibc >= 2.17 and ARM architecture (aarch64) wheels require glibc >= 2.31 (only ARM Server Base System Architecture is supported).
+> <i class="fas fa-microchip text-white"></i> **Architecture:** One of the following architectures:<br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> x86_64 <br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> **[ARM SBSA](https://developer.arm.com/documentation/102378/0201/Armv8-x-A-and-the-SBSA){: target="_blank"}**.  Jetson support coming soon!
+
+> <i class="fas fa-info-circle text-white"></i> **glibc version:**  <br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> x86_64: glibc >= 2.17 <br>
+>> <i class="fas fa-chevron-circle-right text-white"></i> ARM SBSA (aarch64): glibc >= 2.31
 
 > <i class="fas fa-microchip text-white"></i> **GPU:** Only GPUs with **[Compute capability](https://developer.nvidia.com/cuda-gpus){: target="_blank"}** 6.0 or higher (i.e. Pascal generation or newer) are supported.
 
 > <i class="fas fa-download text-white"></i> **CUDA >= 11.5**, with at least the v495.29.05 driver. To use CUDA 11.2, 11.3, or 11.4, please see Troubleshooting and Known Issues.
 
 > <i class="fab fa-python text-white"></i> **Python and pip version:** Python 3.8 or 3.9 using pip 20.3+ with **[PEP600 support](https://peps.python.org/pep-0600/){: target="_blank"}**.
 
+<br>
 ## <i class="far fa-comments text-white"></i> Connect
```

Comment:
Consider breaking connect into a different colored section below as a call to action.